### ISSUE TITLE
To reproduce results for large-sized image datasets (FLICKR, NUS-WIDE)

### ISSUE THREAD

**Issue by solucky-95** [youngkyunJang/SPQ#10]
Hi there,
Thank you for sharing this awesome code. The performance on CIFAR is great, however, the performance on FLICKR and NUS-WIDE seems to be quite low (e.g., ~0.67 on FLICKR 32bit vs. ~0.75 reported on the paper). Do you have any suggestions about how to reproduce your results on large-sized image datasets (e.g., FLICKR, NUS-WIDE)? Any specific hyper-parameter settings?
Thanks.

**Comment by youngkyunJang** [comment]
Hello! Thanks for asking. How do you set image resolutions in your experiments? 
In my setup, I set it as 64 x 64 with bicubic operation. :)

**Comment by solucky-95** [comment]
Thanks. I have tried 64x64 resolutions but still cannot reproduce the reported results (FLICKR25K 32bit mAP@5000 ~0.65 vs. ~0.75).
FLICKR25K is downloaded from the official website and 5000/2000/23000 training/query/retrieval images are randomly selected.
The used hyper-parameters are: image size 64x64, batch size 256, standard ResNet-50 (output feature dim 2048) without ImageNet pre-trained weights, Adam optimizer with lr 3e-4 and cosine scheduling, over 2000 epochs, evaluate every 50 epochs, temperature parameters 5.0 & 0.5.
Are there any specific settings or data splits in your experiments? 1-2% is reasonable but 10% is a large margin.

**Comment by youngkyunJang** [comment]
How did you set the batch size? I think there might be some difference in the train / query configuration. I will share the setup that I used soon!

**Comment by youngkyunJang** [comment]
Sure! I will share them soon :)

**Comment by yayaYsmileT** [comment]
Hello, I would like to know how to achieve the effect of the paper on the CIFAR10 dataset, or how to divide and parameterize the dataset? I can't get mAP
